static/app/views/eventsV2/results.tsx

```diff
@@ -81,12 +81,10 @@ function readShowTagsState() {
   return value === '1';
 }
 
-function getYAxis(location: Location, eventView: EventView, savedQuery?: SavedQuery) {
+function getYAxis(location: Location, savedQuery?: SavedQuery) {
   return location.query.yAxis
     ? decodeList(location.query.yAxis)
-    : savedQuery?.yAxis && savedQuery.yAxis.length > 0
-    ? decodeList(savedQuery?.yAxis)
-    : [eventView.getYAxis()];
+    : decodeList(savedQuery?.yAxis);
```

Comment:
Here we force the y-axis value array to always have at least 1 value, so when the user de-selects the last option, nothing happens:
https://user-images.githubusercontent.com/44172267/167710492-d2d69602-1744-40bf-9694-39df2a139e66.mov

This looks and feels broken. I think it's fine to have no selected value here, especially since we've already implemented an empty state for the chart component:
<img width="906" alt="image" src="https://user-images.githubusercontent.com/44172267/167710875-64cc912b-d929-4b63-b9a3-89eba2eac192.png">